betelgeuse.py

```diff
@@ -644,9 +644,18 @@ def test_case(context, path, collect_only, project):
     '--parent-name',
     help='Name of parent Test Plan to link to.',
 )
+@click.option(
+    '--status',
+    help='Status of the Test Plan',
+    type=click.Choice([
+        'open',
+        'inprogress',
+        'done',
+    ])
+)
 @click.argument('project')
 @click.pass_context
-def test_plan(context, name, plan_type, parent_name, project):
+def test_plan(context, name, plan_type, parent_name, status, project):
```

Comment:
If status is a custom field then it should be defined that way.

Also the custom fields for test run was created so it can be easily reused, if that is not the case we need to improve it.